src/text/mod.rs

```diff
@@ -80,6 +80,45 @@ impl<C> DecorationColor<C> {
     }
 }
 
+/// Text line height.
+///
+/// The line height is defined as the vertical distance between the baseline of two adjacent lines
+/// of text.
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub enum LineHeight {
+    /// Absolute line height in pixels.
+    Pixels(u32),
+
+    /// Relative line height in percent of the default line height.
+    Percent(u32),
+}
+
+impl LineHeight {
+    /// Converts the line height to an absolute pixel distance.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use embedded_graphics::text::LineHeight;
+    ///
+    /// let relative_height = LineHeight::Percent(150);
+    /// assert_eq!(relative_height.to_absolute(20), 30);
+    /// ```
+    pub fn to_absolute(self, base_line_height: u32) -> u32 {
+        match self {
+            Self::Pixels(px) => px,
+            Self::Percent(100) => base_line_height,
```

Comment:
I'm quite sure this special case is not necessary.

Reply:
That was a weird performance optimization, but `to_absolute` is called so infrequently that this probably doesn't make any sense. I'll remove it.